src/test/ui/issues/issue-20692.stderr

```diff
@@ -1,24 +1,25 @@
 error[E0038]: the trait `Array` cannot be made into an object
   --> $DIR/issue-20692.rs:7:5
    |
-LL | trait Array: Sized {}
-   |              ----- the trait cannot require that `Self : Sized`
+LL | trait Array: Sized + Copy {}
+   |              -----   ---- the trait cannot require that `Self : Sized`
```

Comment:
why is there a `Self : Sized` error on `Copy`?

Reply:
This change points at all the super traits that cause a `Sized` obligation and `Copy` is one of them. Removing `Self: Sized` would still cause the error to trigger, but it wouldn't provide a span if we didn't point at `Copy`.